### PR TITLE
README fixes

### PR DESCRIPTION
**Pull request by likamrat**

### CODE REVIEW

File: docs/azure_jumpstart_arcbox/flavors/Full/_index.md [hunk 1]
```diff
@@ -104,21 +104,33 @@ ArcBox uses an advanced automation flow to deploy and configure all necessary re
   az provider register --namespace Microsoft.AzureArcData --wait
   ```
 
-* Create Azure service principal (SP)
+- Create Azure service principal (SP). To deploy ArcBox, an Azure service principal assigned with multiple is required:
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/flavors/Full/_index.md:107]:
```suggestion
- Create Azure service principal (SP). To deploy ArcBox, an Azure service principal assigned with multiple RBAC roles is required:
```

File: docs/azure_jumpstart_arcbox/flavors/ITPro/_index.md [hunk 2]
```diff
@@ -27,69 +27,81 @@ ArcBox deploys several management and operations services that work with ArcBox'
...
 
   ```shell
   az --version
   ```
 
-* Login to AZ CLI using the ```az login``` command.
+- Login to AZ CLI using the ```az login``` command.
 
-* Create Azure service principal (SP)
+- Create Azure service principal (SP). To deploy ArcBox, an Azure service principal assigned with multiple is required:
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_arcbox/flavors/ITPro/_index.md:78]:
```suggestion
- Create Azure service principal (SP). To deploy ArcBox, an Azure service principal assigned with multiple RBAC roles is required:
```

**Comment by dkirby-ms** [comment]
Why move the images to an /img folder? Typically we have kept these at the root path where the index file is. Maybe I am missing something?

**Review by dkirby-ms — CHANGES_REQUESTED** [review]

**Comment by likamrat** [comment]
> 

cleaner

**Pull request merged**